scripts/py_matter_yamltests/matter_yamltests/definitions.py

```diff
@@ -149,6 +161,21 @@ def get_type_by_name(self, cluster_name: str, target_name: str):
 
         return None
 
+    def get_command_names(self, cluster_name: str) -> list[str]:
```

Comment:
should these return sets rather than lists?

Reply:
I would prefer to keep the order. This things is meant to be displayed and it may be confusing to have it ordered differently times to times.